### ISSUE TITLE
One command

### ISSUE THREAD

**Issue by proxyblue** [phynet/SSCodesigningXcode#1]
Can you modify the ruby script so that everything works in one command with arguments rather than with prompts? 

It's okay if everything is an absolute path.

**Comment by phynet** [comment]
Hi, @louiscremen. I'm working on it, I need to test it and I'll pull it this week, and I will also put an option to show all Certificates and Provisioning Profiles. Thanks for your comment

**Comment by proxyblue** [comment]
Thank you for replying - if I get some time (and a bit of Ruby google searching) - I might be able to add it in myself. I tried piping the data through with \n but chomp doesn’t like me doing that.

Thanks, this repo has appeared around the same time that I needed this functionality. Good timing :)

**Comment by phynet** [comment]
Cool, well, I will try to upload it tonight then (Spain Time) so you can use it. :8ball:

**Comment by phynet** [comment]
Hello @louiscremen ... I've updated the files. So now you can use CL in one line. Something to have in mind is that is mandatory now to pass the **path of Provisioning Profile**, because it changes from every user (I don't know yet how to implement this without making the user pass it in CL). 

Also I've added two new options to list certificate profiles and provisioning profiles stored in your mac. Right now you can use it like: 

```
ruby main.rb cert
```

or 

```
ruby main.rb prov /Users/yourUser/Library/MobileDevice/Provisioning\ Profiles/
```

Maybe I could solve it with relative paths...I don't know yet. I need more time to improve all this code. xD 

So, in conclusion you can use the command like this:

```
ruby main.rb "The Name of your certificate" "Name of your provisioning profile" /Users/YourUser/Path-to-your-project-xcode   /Users/YourUser/Library/MobileDevice/Provisioning\ Profiles/ 
```

I hope this works for you. Give me some feed back so I could change something.

Cheers.
